ext/standard/array.c

```diff
@@ -6500,7 +6500,7 @@ PHP_FUNCTION(array_filter)
 	zval args[2];
 	zval retval;
 	bool have_callback = 0;
-	zend_long use_type = 0;
+	zend_long mode = ARRAY_FILTER_USE_VALUE;
```

Comment:
Please avoid the rename.

Reply:
Isn't mode a more relevant name than `use_type` that suggests boolean value? 

This only touches internal variable, so I thought that shouldn't be a problem.

Reply:
I think `use_type` refers to "ARRAY_FILTER_**USE_**", so `mode` doesn't seem more descriptive.

Reply:
`mode` is the literal name of the argument in the `array_filter` function. 

https://www.php.net/manual/en/function.array-filter.php

For me, it doesn't make a big difference, rather than alignment, so I can change it tomorrow.